src/qt/intro.cpp

```diff
@@ -133,9 +133,9 @@ Intro::Intro(QWidget *parent, uint64_t blockchain_size, uint64_t chain_state_siz
         ui->prune->setChecked(true);
         ui->prune->setEnabled(false);
     }
-    const int prune_target_gb = PruneMiBtoGB(prune_target_mib);
-    ui->prune->setText(tr("Discard blocks after verification, except most recent %1 GB (prune)").arg(prune_target_gb ? prune_target_gb : DEFAULT_PRUNE_TARGET_GB));
-    UpdatePruneLabels(prune_target_gb);
+    m_prune_target_gb = prune_target_mib ? PruneMiBtoGB(prune_target_mib) : DEFAULT_PRUNE_TARGET_GB;
```

Comment:
In commit "gui: Fix intro dialog when prune is toggled" (f07dd7c69ed7a5245f79662100c859286557b73b)

I think this should say `prune_target_mib > 1 ? ...` instead of `prune_target_mib ? ...`

If the pruning value is 1, pruning is still effectively disabled and more than 1GB will be required unless manual pruning is done.

Could also combine this check into the `if (prune_target_mib > 1)` check above.

Reply:
Fixed in the latest push.